src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Book.ttf) format("truetype");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Heavy.ttf) format("truetype");
+  font-style: normal;
+  font-weight: 800;
+}
+
+:root {
+  --main-bg-color: #e1e7ed;
+  --main-bg-gradient: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+body {
+  margin-top: 20px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.input::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+}
+
+.input {
+  font-family: "Avenir", georgia, serif;
+  vertical-align: top;
+  outline: none;
+  border: 1px solid var(--main-bg-color);
+  width: 100%;
+  border-radius: 4px;
+}
+
+.input:hover {
+  border: 1px solid var(--main-bg-color);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  font-family: "Avenir", georgia, serif;
+  font-weight: 800;
+  vertical-align: top;
+  border: 1px solid var(--main-bg-color);
+  background-color: var(--main-bg-gradient);
+  border-radius: 3px;
+}
+
+.bigInput {
+  font-size: 16px;
+  height: 70px;
+  padding: 0 0 0 62px;
+  margin-bottom: 20px;
+  background: url(images/Search.svg) no-repeat 26px;
+  background-size: 19px 19px;
+}
+
+.bigInput :hover {
+  background-size: 19px 19px;
+}
+
+.bigInput :focus {
+  font-size: 16px;
+  background: url(images/Search.svg) no-repeat 26px;
```

Comment:
the gradient still doesn't work
try to google how to do it or ask in the chat if you need help

example: https://www.geeksforgeeks.org/css-combine-background-image-with-gradient-overlay/